### PR TITLE
Build configs

### PR DESCRIPTION
**Pull request by sedroche**
Tickets: https://issues.jboss.org/browse/FH-4154, https://issues.jboss.org/browse/FH-4155

New screens for mobile app builds.
- build config tab with create, edit, view modes
- mobile builds tab to display the app build
- modal fixes

### CODE REVIEW

File: ui/public/scripts/components/mobile-app/enable-mobile-build.component.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                <span class="pficon pficon pficon-add-circle-o"></span>
+              </div>
+              <h1>
+                Enable Mobile CI/CD Service
+              </h1>
+              <p>
+                To enable mobile application builds, please provision the Mobile CI/CD service.
+              </p>
+              <p>
+                This can be provisioned via the application catalog.
```
Comment (david-martin) [inline ui/public/scripts/components/mobile-app/enable-mobile-build.component.js]: 'Service Catalog' is the term used elsewhere in OpenShift

File: ui/public/scripts/components/mobile-app/enable-mobile-build.component.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+              </p>
+              <p>
+                Learn more about this <a href="http://feedhenry.org/docs/">in the documentation</a>.
+              </p>
+              <div class="blank-slate-pf-main-action">
+                <a ng-href="/" class="btn btn-primary btn-lg">Application Catalog</a>
```
Comment (david-martin) [inline ui/public/scripts/components/mobile-app/enable-mobile-build.component.js]: This button might read better as 'Provision Mobile CI/CD Service'

File: ui/public/scripts/components/mobile-app/view-app-config.component.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+ * # view-app-config
+ */
+angular.module('mobileControlPanelApp').component('viewAppConfig', {
+  template: `<div>
+              <div class="row app-config">
+                <div class="col-lg-6">
+                  <h3>Details</h3>
+                  <dl class="dl-horizontal left">
+                    <div>
+                      <dt>Name</dt>
```
Comment (david-martin) [inline ui/public/scripts/components/mobile-app/view-app-config.component.js]: Could we use 'Jenkins Job Name' here, and move the input down the form? It's lower priority than the Repo details

File: ui/public/scripts/components/mobile-app/view-app-config.component.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+/**
+ * @ngdoc component
+ * @name mcp.component:view-app-config
+ * @description
+ * # view-app-config
+ */
+angular.module('mobileControlPanelApp').component('viewAppConfig', {
+  template: `<div>
+              <div class="row app-config">
```
Comment (david-martin) [inline ui/public/scripts/components/mobile-app/view-app-config.component.js]: Could we add some inline help text (beside/below) each of these form inputs?

Comment (sedroche) [reply ui/public/scripts/components/mobile-app/view-app-config.component.js]: What sort of help text to you envision here? Each value has a descriptive label and we get here from the create screen which has inline help and validation

File: ui/public/scripts/components/mobile-app/view-app-config.component.js (hunk 3 above)
Comment (sedroche) [reply ui/public/scripts/components/mobile-app/view-app-config.component.js]: Have updated this here and on the other edit/create views

File: ui/public/scripts/components/mobile-app/enable-mobile-build.component.js (hunk 2 above)
Comment (sedroche) [reply ui/public/scripts/components/mobile-app/enable-mobile-build.component.js]: Updated

File: ui/public/scripts/components/mobile-app/view-app-config.component.js (hunk 4 above)
Comment (david-martin) [reply ui/public/scripts/components/mobile-app/view-app-config.component.js]: @sedroche Nevermind. I realise my mistake as this is the viewAppConfig. I thought this was the Create/Edit app config.

**Pull request merged**